fake_ubersmith/api/ubersmith.py

```diff
@@ -15,11 +18,35 @@ def hook_to(self, server):
             methods=["POST"]
         )
 
+        self.register_endpoints(
+            ubersmith_method='hidden.enable_crash_mode',
+            function=self.enable_crash_mode
+        )
+        self.register_endpoints(
+            ubersmith_method='hidden.disable_crash_mode',
+            function=self.disable_crash_mode
+        )
+
+    def enable_crash_mode(self, form_data):
+        self.crash_mode = True
+        return response(data="Crash Mode Enabled")
+
+    def disable_crash_mode(self, form_data):
+        self.crash_mode = False
+        return response(data="Crash Mode Disabled")
+
     def register_endpoints(self, ubersmith_method, function):
         self.methods[ubersmith_method] = function
 
+    def _should_crash(self, method):
+        return method not in ['hidden.enable_crash_mode', 'hidden.disable_crash_mode'] and self.crash_mode
+
     def _route_method(self):
-        return self.methods[request.form['method']](request.form)
+        method = request.form['method']
+        if self._should_crash(method):
+            raise Exception("Crash mode was enabled")
```

Comment:
We shouldn't raise a generic Exception here. At least a FakeUbersmithError imho.